src/envoy/http/authn/filter_context.cc

```diff
@@ -102,15 +102,17 @@ bool FilterContext::getJwtPayloadFromEnvoyJwtFilter(const std::string& issuer,
     return false;
   }
 
-  // Serialize the payload from Envoy jwt filter first before writing it to |payload|.
-  // TODO (pitlv2109): Return protobuf Struct instead of string, once Istio jwt filter is removed.
-  // Also need to change how Istio authn filter processes the jwt payload.
+  // Serialize the payload from Envoy jwt filter first before writing it to
```

Comment:
nit: why are all these changes showing up here? are we using a different formatting ?

Reply:
I ran scripts/run_gofmt.sh that we are using in istio.io/istio... looks like it was added only 7 days back.. I can revert these changes if we don't intend to use this script for formatting?

Reply:
`make check` is broken (see #2287), so the last few commits were committed without formatting being properly enforced.

Reply:
Fixed, please merge with `master`.

Reply:
Thanks @PiotrSikora